build.sbt

```diff
@@ -169,6 +169,8 @@ lazy val server = (project in file("server"))
 
       "com.google.code.findbugs" % "jsr305" % "3.0.2",
       "com.h2database" %  "h2" % "2.2.224",
+      "com.mysql" % "mysql-connector-j" % "8.4.0",
```

Comment:
Why not use the latest driver? https://mvnrepository.com/artifact/com.mysql/mysql-connector-j

Reply:
Thanks for the comment! I didn't realize there's 9.0.0 release when I started. It's now updated.